### ISSUE TITLE
"strange result" with substitute no longer strange

### ISSUE THREAD

**Issue by foxjwill** [Macaulay2/M2#695]
The [documentation for `substitute`](https://faculty.math.illinois.edu/Macaulay2/doc/Macaulay2-1.10/share/doc/Macaulay2/Macaulay2Doc/html/_substitute.html) says "If the values of all of the variables are in a different ring, then the result will be in that ring." Then below this there's some output, but this output is the same as the output for "By changing one of the values to a rational number, we insure that the result will be rational."

And while we're here, the correct word to use is "ensure", not "insure".

**Comment by DanGrayson** [comment]
Thanks for both catches!  Here is the relevant bit:
```

      If the values of all of the variables are in a different ring, then the result will be in that ring.

      +----------------------------------------+
      |i10 : use ring F;                       |
      +----------------------------------------+
      |i11 : sub(F, {a=>1, b=>3, c=> 1, d=>13})|
      |                                        |
      |o11 = 88                                |
      |                                        |
      |o11 : QQ                                |
      +----------------------------------------+

      This can have strange results, if the values are all integers, but fractions are present.

      +--------------------------------------------+
      |i12 : sub(1/3*a*b, {a=>1, b=>1, c=>1, d=>1})|
      |                                            |
      |      1                                     |
      |o12 = -                                     |
      |      3                                     |
      |                                            |
      |o12 : QQ                                    |
      +--------------------------------------------+

      By changing one of the values to a rational number, we insure that the result will be rational.

      +------------------------------------------------------+
      |i13 : sub(1/3*a*b, {a=>1_QQ, b=>1, c=>1, d=>1})       |
      |                                                      |
      |      1                                               |
      |o13 = -                                               |
      |      3                                               |
      |                                                      |
      |o13 : QQ                                              |
      +------------------------------------------------------+
```

Part is wrong, too, since we changed the noxious behavior being documented.  We'll have to come up with better examples.  I think what we made it do is to replace the ring containing the values by one minimally large enough to sensibly receive a ring homomorphism from the coefficient ring of the polynomial ring.

**Comment by d-torrance** [comment]
This was fixed way back in #870.
